packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -61,29 +94,39 @@ export const UnitRow = ({
         {children}
       </div>
 
-      <div className="unit-row-actions">
-        <div>
-          {route && (
-            <Link
-              className="cta-neutral cta-base"
-              data-testid="unit-row-route"
-              to={`${route}${location.search}`}
-            >
-              {ctaText}
-            </Link>
-          )}
+      <div className="flex items-center">
```

Comment:
This is what we decided because of the button breakage on mobile and certain wide desktop widths, I'm happy to move it back so long as it doesn't break again

Reply:
It appears to be working well now, had to remove some `ml-2` classes on the buttons we added when trying to get it working before.

Reply:
I'm not fully up to date on the specific issue, but if it persists with that class feel free to ping me with a description of what's going on and I'll see what I can do -- I love that kind of stuff

Reply:
It looks like this change resulted in a wonky layout for desktop:

![Screen Shot 2020-09-09 at 1 35 03 PM](https://user-images.githubusercontent.com/6392049/92634421-4ab1a000-f2a2-11ea-98c3-6a4d68168d68.png)

And while there's nothing visually wrong with this mobile layout, it doesn't match the Figma doc:

![Screen Shot 2020-09-09 at 1 34 57 PM](https://user-images.githubusercontent.com/6392049/92634438-51d8ae00-f2a2-11ea-814e-292ef18801be.png)

However, this looks like a relatively easy fix. Check out this patch, it should do the trick:

```diff
diff --git a/packages/fxa-settings/src/components/UnitRow/index.tsx b/packages/fxa-settings/src/components/UnitRow/index.tsx
index f0687213b..0b43a30ed 100644
--- a/packages/fxa-settings/src/components/UnitRow/index.tsx
+++ b/packages/fxa-settings/src/components/UnitRow/index.tsx
@@ -13,6 +13,7 @@ type ModalButtonProps = {
   revealModal: () => void;
   modalRevealed?: boolean;
   alertBarRevealed?: boolean;
+  leftSpaced?: boolean;
 };
 
 export const ModalButton = ({
@@ -21,6 +22,7 @@ export const ModalButton = ({
   revealModal,
   modalRevealed,
   alertBarRevealed,
+  leftSpaced,
 }: ModalButtonProps) => {
   const modalTriggerElement = useRef<HTMLButtonElement>(null);
   // If the UnitRow children contains an AlertBar that is revealed,
@@ -35,8 +37,9 @@ export const ModalButton = ({
   return (
     <button
       className={classNames(
-        'cta-base cta-neutral transition-standard mobileLandscape:ml-2',
-        className
+        'cta-base transition-standard',
+        leftSpaced && 'ml-2',
+        className || 'cta-neutral'
       )}
       data-testid="unit-row-modal"
       ref={modalTriggerElement}
@@ -81,6 +84,7 @@ export const UnitRow = ({
   ctaText = ctaText || (headerValue ? 'Change' : 'Add');
 
   const location = useLocation();
+  const multiButton = !!(route || secondaryCtaRoute)
 
   return (
     <div className="unit-row">
@@ -98,10 +102,10 @@ export const UnitRow = ({
       </div>
 
       <div className="unit-row-actions">
-        <div>
+        <div className="flex items-center">
           {route && (
             <Link
-              className="cta-neutral cta-base transition-standard mobileLandscape:mr-2"
+              className="cta-neutral cta-base transition-standard mr-2"
               data-testid="unit-row-route"
               to={`${route}${location.search}`}
             >
@@ -110,12 +114,13 @@ export const UnitRow = ({
           )}
 
           {revealModal && (
-            <ModalButton {...{ revealModal, ctaText, alertBarRevealed }} />
+            <ModalButton
+              leftSpaced={multiButton} {...{ revealModal, ctaText, alertBarRevealed }} />
           )}
 
           {secondaryCtaRoute && (
             <Link
-              className="cta-neutral transition-standard mobileLandscape:mr-2"
+              className="cta-neutral cta-base transition-standard mr-2"
               data-testid="unit-row-route"
               to={`${secondaryCtaRoute}${location.search}`}
             >
@@ -125,6 +130,7 @@ export const UnitRow = ({
 
           {revealSecondaryModal && (
             <ModalButton
+              leftSpaced={multiButton}
               revealModal={revealSecondaryModal}
               ctaText={secondaryCtaText}
               className={secondaryButtonClassName}
```

Reply:
So weird this wasn't showing up for me in storybook, I wonder if I need to be testing on a second monitor, or adjusting my resolution for retina screens. Anyways, I made the changes from the patch and it's looking good in all the states on my machine.